### DISCUSSION TITLE
React Support

### DISCUSSION THREAD

**Discussion by tderflinger** [GrapesJS/grapesjs#3830] [Ideas]
Hello GrapeJs! 

Thanks for this great open source editor! As a React developer I am looking for ways to speed up development of React components. So I want to ask if support for exporting React components is something you are considering?
Is this on your roadmap or out of scope?

Thanks for your opinion!
-Tommy

**Comment by artf** [comment]
It's already possible, but we don't have yet the official React renderer (so it might require a bit of work from your side), check my demo posted here: https://github.com/artf/grapesjs/issues/1970#issuecomment-905751478
